hammer/library/jiraoperations.py

```diff
@@ -17,20 +17,25 @@
 
 class JiraReporting(object):
     """ Base class for JIRA reporting """
-    def __init__(self, config):
+    def __init__(self, config, module=''):
         self.config = config
-        self.jira = JiraOperations(self.config)
+        self.jira = JiraOperations(self.config, module=module)
+        self.module_jira_enabled = getattr(config, module).jira if hasattr(config, module) else True
 
+    def jira_enabled(func):
```

Comment:
Can we move this decorator out of the class?
It looks a bit unusual (we pass func and not self) and we don't use it as a method of a class.

Reply:
I think it will be better to change it to 'private' method

Reply:
what is private method?

Reply:
New commit has been added